cmd/kubeadm/app/phases/addons/dns/versions.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dns
+
+import (
+	"k8s.io/kubernetes/pkg/util/version"
+)
+
+const (
+	kubeDNSv170AndAboveVersion = "1.14.4"
```

Comment:
We need the Real Addon Manager (tm) here. We don't have that unfortunately.
That combined with HA is probably some of the next step for the SIG, but need to get this now for upgrades shorter-term.